addon/pods/components/rui-dropdown-trigger/component.js

```diff
@@ -6,8 +6,13 @@ export default Ember.Component.extend({
   tagName: 'button',
   classNames: ['dropdown-toggle'],
   classNameBindings: ['computedStyle'],
-  attributeBindings: ['type', 'data-toggle', 'haspopup', 'aria-expanded'],
-  id: '',
+  attributeBindings: [
+    'type',
+    'data-toggle',
+    'haspopup',
```

Comment:
Likely from here.  Not sure why the aria part didn't make it in.
http://v4-alpha.getbootstrap.com/components/dropdowns/#example